pages/article/[articleLink].js

```diff
@@ -111,12 +111,12 @@ const Article = () => {
                   <h3 className="py-3 font-yrsa text-lg font-medium uppercase tracking-widest shadow-neutral-900">
                     Detaily
                   </h3>
-                  {Object.entries(details).map(({ detailId, value }) => (
-                    <div key={detailId} className="bg-stone-100 p-2">
+                  {Object.entries(details).map((item) => (
```

Comment:
As I have pointed in multiple comments in multiple PR :D. Consider doing 
```suggestion
                  {Object.entries(details).map(([detailId, value]) => (
```

or any other suiting names